docs: Sandbox play with Mimir (#9247)
grafana/mimir: docs/sources/mimir/get-started/play-with-grafana-mimir/index.md

menuTitle: Play with Mimir
title: Play with Mimir
weight: 10
killercoda:
  title: Play with Mimir
  description: Learn about Grafana Mimir, which provides distributed, horizontally scalable, and highly available long term storage for Prometheus.
  details:
    intro:
      foreground: "docker-compose-update.sh"
  preprocessing:
    substitutions:

>       - regexp: 'tutorial-architecture.png'

Reviewer: I think this should be fine as the linter wants, I think the issue with parsing must have been something else.

I've also escaped the period in the regular expression because we want to match it literally rather than match any character. I believe with YAML double quoted strings we need to escape the backslash

```suggestion
      - regexp: "tutorial-architecture\\.png"
```

- Run Grafana to explore Grafana Mimir dashboards
- Configure a testing recording rule and alert in Grafana Mimir

<!-- INTERACTIVE ignore START -->

## Prerequisites

- Git
- [Docker](https://docs.docker.com/) and [Docker Compose](https://docs.docker.com/compose/)
- Availability of both ports `9000` and `9009` on your host machine

{{< admonition type="tip" >}}
Alternatively, you can try out this example in our interactive learning environment: [Play with Mimir](https://killercoda.com/grafana-labs/course/mimir/play-with-mimir).

> It's a fully configured environment with all the dependencies already installed.

Reviewer 2: ```suggestion
It's a fully configured environment with all dependencies already installed.
```

   ```
1. Confirm the query returns a value of `3` which is the number of Mimir instances currently running in your local setup.

<!-- INTERACTIVE page step4.md END -->

<!-- INTERACTIVE page step5.md START -->

## Configure your first alert rule

Alerting rules allow you to define alert conditions based on PromQL expressions and to send notifications about firing
alerts to Grafana Mimir Alertmanager. In this section you're going to configure an alerting rule in Grafana Mimir using
tooling offered by Grafana.

1. Open [Grafana Alerting](http://localhost:9000/alerting/list).
1. Click **New alert rule**.
1. Configure the alert rule:
   1. Type `MimirNotRunning` in the **Rule name** field.
   1. Choose **Mimir** in the **Select data source** field.

>    1. Type `count(up == 0)` in the **Metrics browser** query field. This will currently show `no data` since all instances are running.

Reviewer 2: ```suggestion
   1. Type `count(up == 0)` in the **Metrics browser** query field. This currently shows `no data` since all instances are running.
```

1. Scroll down to **Set evaluation behavior**:
   1. Select `New folder` and type `example-folder` in the **Folder name** field.

>    1. Select `New evaluation group` and type `example-group` in the **Group name** field. Set evaluation interval to `30s`.

Reviewer 2: ```suggestion
   1. Select `New evaluation group` and type `example-group` in the **Group name** field. Set the evaluation interval to `30s`.
```
